cmake/onnxruntime_providers.cmake

```diff
@@ -696,7 +695,7 @@ if (onnxruntime_USE_TENSORRT)
   endif()
 
   include_directories(${TENSORRT_INCLUDE_DIR})
-
+  MESSAGE(STATUS "Found TensorRT libs at ${TENSORRT_LIBRARY}")
```

Comment:
If the build goes to oss parser path (not built-in parser), the variable TENSORRT_LIBRARY will be empty. The message might show "Found TensorRT libs at", I think it's a little bit confusing for user.

Reply:
it's not empty. we rely on TENSORRT_LIBRARY to be populated in both paths. otherwise, we can't build TRT EP.

Reply:
note in line 699 below we set trt_link_libs to ${TENSORRT_LIBRARY}

Reply:
for the oss parser path, onnx-tensorrt populates the TENSORT_LIBRARY